Tests/LibWeb/Ref/reference/webkit-text-fill-color.html

```diff
@@ -0,0 +1,9 @@
+<style>
+    * {
+      margin: 0;
+    }
+    body {
+      background-color: white;
+    }
+</style>
+<img src="./images/webkit-text-fill-color.png">
```

Comment:
This reference page can be done with the `color` property instead. It's preferred to not use a screenshot like this unless it's necessary.

Reply:
`color` alone won't work, but together with `text-decoration-color:black` it should be possible. Our support for `text-decoration-color` isn't perfect, but should be good enough for this.

Reply:
Oh right, I reviewed this on mobile and couldn't see the screenshot. :man_facepalming: You're right.

Reply:
changed to not use an image